src/io/flutter/view/FlutterView.java

```diff
@@ -296,9 +350,7 @@ public final void update(AnActionEvent e) {
 
 class ObservatoryTimelineAction extends AbstractFlutterAction {
   ObservatoryTimelineAction(@NotNull FlutterView view) {
-    super(view, FlutterBundle.message("flutter.view.observatoryTimeline.text"),
-          FlutterBundle.message("flutter.view.observatoryTimeline.description"),
-          AllIcons.Debugger.Value);
+    super(view, "Open Observatory Timeline");
```

Comment:
I assume we no longer want to pass the `AllIcons.Debugger.Value` icon up to super?